cli/src/command/generate_config.rs

```diff
@@ -128,18 +128,21 @@ impl ExonumCommand for GenerateConfig {
 
         let public_config_path = self.output_dir.join(PUB_CONFIG_FILE_NAME);
         let secret_config_path = self.output_dir.join(SEC_CONFIG_FILE_NAME);
-        let master_key_path = self
+        let mut master_key_path = self
             .master_key_path
-            .unwrap_or_else(|| "".into())
+            .unwrap_or_default()
             .join(MASTER_KEY_FILE_NAME);
 
+        if !master_key_path.is_absolute() {
```

Comment:
:+1: Just a nit: I didn't know, but it looks like `join` will properly work with absolute paths. [Docs](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.push) say that the path will be replaced by the argument if the latter one is absolute.

Reply:
Yes, from doc: `Pushing an absolute path replaces the existing path:`.